creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -77,6 +77,7 @@ export default async function init(el) {
     case el.classList.contains('firefly'): {
       interactiveInit(el);
       loadStyle('/creativecloud/features/interactive-elements/interactive-elements.css');
+      loadStyle('/creativecloud/features/firefly/firefly-interactive.css');
```

Comment:
The relatiev path would cause problem with fragment rendering on AEM. We need to make all the relative paths to start with getConfig() from Milo and use following to make the load path 
const origin = conf.origin || window.location.origin;
We can possibly create a funciton in utils that we can export when required.

Reply:
Confirmed with Amanda, this block is not going to be used in AEM